templates/legal/terms-and-policies/cube-terms.html

```diff
@@ -8,149 +8,170 @@
-    </p>
-    <h2>7. Liability</h2>
-    <p>
-        Your use of the Service is at your sole risk. The Service is provided on an “as is” and “as available” basis without warranty of any kind.
-    </p>
-    <p>
-        In the case of the Service provided by Canonical, the Service is provided “as is” and, other than as expressly set out in these Terms of Service, all warranties (whether express, implied, statutory or otherwise) in respect of the Service are expressly excluded to the maximum extent permitted by law
-    </p>
-    <p>
-        Canonical will provide the Service with reasonable care and skill, and Canonical will use reasonable efforts to ensure the availability of the Services, but makes no guarantee that the Service will be available without interruption or will be error-free.
-    </p>
-    <p>
-        In the case of third party content, you acknowledge that such content is provided to you by the relevant third party pursuant to the relevant terms and conditions and Canonical will have no liability whatsoever in contract, tort or otherwise to you in respect of such content.
-    </p>
-    <p>
-        Canonical will not be liable in contract, tort or otherwise for any: indirect or consequential loss; loss of profits; loss of revenue; loss of anticipated savings; loss of business or business opportunity; loss of goodwill; or loss of or corruption to data. Otherwise, Canonical’s total liability in contract, tort or otherwise for any claims is limited to £10.
-    </p>
-    <p>
-        Nothing in these Terms of Service will exclude or limit Canonical’s liability for: death or personal injury caused by the negligence of Canonical; fraud or fraudulent misrepresentation; or any other liability that cannot be excluded or limited by law.
-    </p>
-    <h2>8. General</h2>
-    <p>
-        These Terms of Service are governed by the laws of England and any dispute will be heard by the courts in England. 
-    </p>
-    <p>
-        Failure by Canonical to enforce any right or provision of these Terms of Service shall not constitute a waiver of such right or provision. If any part of these Terms of Service is held invalid or unenforceable, that part will be construed to reflect the parties original intent, and the remaining portions will remain in full force and effect. The terms of these Terms of Service do not affect your statutory rights.
-    </p>
-    <p>
-        Any notices should be sent by registered post to:
-    </p>
-    <p>Legal, Canonical<br>
-        5th Floor, Blue Fin Building<br>
-        110 Southwark Street<br>
-        London, SE1 0SU<br>
-        United Kingdom
-    </p>
-</div>
-</div>
+<section class="p-strip row">
+    <div class="row">
+        <div class="col-8">
+            <h1>CUBE Terms of Service</h1>
+            <p class="p-heading--three">
+                Valid since July 2021
+            </p>
+            <div class="col-8">
```

Comment:
We've got the col-8 on line 13 so don't need this one. Make sure you check the closing div tags that everything is open and closed correctly